README.md

```diff
@@ -55,98 +62,110 @@ source $HOME/.wasmedge/env
 > During the setup procedure, make sure to select `Add LLVM to the system PATH for all users` or `for the current user`.
 
 2. Restart your PC, or log out and log back in, which allows the LLVM path to be properly
-    * Alternatively you can add the LLVM path `C:\Program Files\LLVM\bin` to your system PATH.
 
-3. Use `moxin-runner` to auto-setup WasmEdge and then build & run Moxin:
+   - Alternatively you can add the LLVM path `C:\Program Files\LLVM\bin` to your system PATH.
+
+3. Use `moly-runner` to auto-setup WasmEdge and then build & run Moxin:
+
 ```sh
-cargo run -p moxin-runner -- cargo run --release  ## `--release` is optional
+cargo run -p moly-runner -- cargo run --release  ## `--release` is optional
 ```
 
----------------------------
+---
 
 ## Packaging Moxin for Distribution
 
-> Note: we already have [pre-built releases of Moxin](https://github.com/moxin-org/moxin/releases) available for download.
+> Note: we already have [pre-built releases of Moxin](https://github.com/moxin-org/moly/releases) available for download.
 
 Install `cargo-packager`:
+
 ```sh
 rustup update stable  ## Rust version 1.79 or higher is required
 cargo +stable install --force --locked cargo-packager
 ```
-For posterity, these instructions have been tested on `cargo-packager` version 0.10.1, which requires Rust v1.79.
 
+For posterity, these instructions have been tested on `cargo-packager` version 0.10.1, which requires Rust v1.79.
 
 ### Packaging for Linux
-On a Debian-based Linux distribution (e.g., Ubuntu), you can generate a `.deb` Debian package, an AppImage, and a pacman installation package.
 
+On a Debian-based Linux distribution (e.g., Ubuntu), you can generate a `.deb` Debian package, an AppImage, and a pacman installation package.
 
 > [!IMPORTANT]
 > You can only generate a `.deb` Debian package on a Debian-based Linux distribution, as `dpkg` is needed.
- 
+
 > [!NOTE]
 > The `pacman` package has not yet been tested.
 
-Ensure you are in the root `moxin` directory, and then you can use `cargo packager` to generate all three package types at once:
+Ensure you are in the root `moly` directory, and then you can use `cargo packager` to generate all three package types at once:
+
 ```sh
 cargo packager --release --verbose   ## --verbose is optional
 ```
 
 To install the Moxin app from the `.deb`package on a Debian-based Linux distribution (e.g., Ubuntu), run:
+
 ```sh
 cd dist/
-sudo apt install ./moxin_0.1.0_amd64.deb  ## The "./" part is required
+sudo apt install ./moly_0.1.0_amd64.deb  ## The "./" part is required
 ```
-We recommend using `apt install` to install the `.deb` file instead of `dpkg -i`, because `apt` will auto-install all of Moxin's required dependencies, whereas `dpkg` will require you to install them manually.
 
+We recommend using `apt install` to install the `.deb` file instead of `dpkg -i`, because `apt` will auto-install all of Moxin's required dependencies, whereas `dpkg` will require you to install them manually.
```

Comment:
```suggestion
We recommend using `apt install` to install the `.deb` file instead of `dpkg -i`, because `apt` will auto-install all of Moly's required dependencies, whereas `dpkg` will require you to install them manually.
```